ajax/libs/fullcalendar-scheduler/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "fullcalendar-scheduler",
+  "filename": "scheduler.min.js",
+  "version": "1.9.0",
+  "description": "A premium add-on to FullCalendar for displaying events and resources",
+  "author": {
+    "name": "Adam Shaw",
+    "email": "arshaw@arshaw.com",
+    "homepage": "http://arshaw.com/"
+  },
+  "homepage": "https://fullcalendar.io/scheduler/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/fullcalendar/fullcalendar-scheduler.git"
+  },
+  "keywords": [
+    "calendar",
+    "event",
+    "full-sized",
+    "jquery-plugin"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/fullcalendar/fullcalendar-scheduler.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      }
+    ]
+  },
+  "licenses": [
+    "GPL-3.0",
+    "CC-BY-NC-ND-4.0",
+    {
+      "name": "fullcalendar-scheduler",
```

Comment:
Let's use `Commercial License`
As this discussion https://github.com/cdnjs/cdnjs/pull/11551#issuecomment-322674874
Also the LICENCE.md show `Commercial License` (https://github.com/fullcalendar/fullcalendar-scheduler/blob/master/LICENSE.md)
Many thanks.